### PR TITLE
Terraform 0.13 support and staging fixes

### PR DESCRIPTION
**Pull request by danielhoherd**
We'll need to change these refs to tags before merging this.

Related to https://github.com/astronomer/issues/issues/3490

### CODE REVIEW

**Comment by danielhoherd** [comment]
@pgvishnuram you can continue working off this branch. I've merged your changes from testistio into this.

**Comment by pgvishnuram** [comment]
yes saw them. I need to add postgres changes as well and run it once locally

**Comment by pgvishnuram** [comment]
@danielhoherd - droneci has been removed so this can be ignored

**Review by pgvishnuram — APPROVED** [review]

File: main.tf [hunk 1]
```diff
@@ -129,7 +133,7 @@ module "system_components" {
 module "astronomer" {
   dependencies = [module.system_components.depended_on, module.gcp.depended_on]
 
-  source = "github.com/astronomer/terraform-kubernetes-astronomer?ref=1.1.90-1"
+  source = "github.com/astronomer/terraform-kubernetes-astronomer?ref=8c20811"
```
Comment (danielhoherd) [inline main.tf:136]: @pgvishnuram I don't think that merging this with just a sha is a good idea. That sha is from a feature branch and will be squashed away when that branch is merged. We should fix https://github.com/astronomer/terraform-kubernetes-astronomer/pull/33#issuecomment-929717662 and update this.

File: provider_workaround.tf [hunk 2]
```diff
@@ -2,24 +2,31 @@
 # initialization of these providers
 # https://github.com/hashicorp/terraform/issues/2430
 
-resource "local_file" "kubeconfig" {
-  content  = module.gcp.kubeconfig
-  filename = "./kubeconfig"
-}
+#data "google_client_config" "default" {}
+
+
```
Comment (danielhoherd) [inline provider_workaround.tf:7]:
```suggestion
```

File: provider_workaround.tf [hunk 3]
```diff
@@ -2,24 +2,31 @@
...
-  load_config_file = true
+  host             = "https://${module.gcp.endpoint}"
+  token            = data.google_client_config.default.access_token
+  cluster_ca_certificate = base64decode(
+    module.gcp.ca_certificate
+  )
 }
 
+
+
```
Comment (danielhoherd) [inline provider_workaround.tf:20]:
```suggestion
```

**Comment by pgvishnuram** [comment]
@danielhoherd terraform-kubernetes-astronomer module bumped with all changes. we are good to merge

**Pull request merged**